pysonnet/projects.py

```diff
@@ -1209,6 +1336,7 @@ def add_output_file(self, file_type, output_folder=None, deembed=True,
 
         if file_name is None:
             file_name = '$BASENAME'
+            self.file_name = file_name
```

Comment:
This file name is the touchstone S parameter file name. I don't think you need that for this PR. Also, why would you want to save it only if it's set to $BASENAME?

Reply:
I don't see where you are saving a string as a file name .son in any of your methods. From what I understand the .ts file should have the same name as the .son file so I thought to save that instead. The only other thing I see is file_string in make_sonnet_file but I dont know what a netlist is

Reply:
Line 409 in this commit is where self.project_file_path is saved. Control-F is useful for navigating large files